javascript/react/intro-react/what-i-need-to-use-react.md

```diff
@@ -17,12 +17,12 @@ aspects:
   - introduction
 
 ---
-# **React** dependencies
+# React dependencies
 
 ---
 ## Content
 
-Being a truly flexible library, you can create applications from scratch with **React**, but also gradually including it to your code base without having to re-write anything.
+Being a truly flexible library, you can create applications from scratch with **React**, but also gradually including it to your codebase without having to re-write anything.
```

Comment:
```suggestion
Being a truly flexible library, you can create applications from scratch with **React**, but also gradually including it to your codebase.
```

The " without having to re-write anything." depends on the situation, sometimes you will have to re-write stuff